modules/lvms-about-volume-clones.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-about-volume-clones_{context}"]
+= About volume clones
+
+A volume clone is a duplicate of an existing persistent volume claim (PVC). You can create a volume clone to make a point-in-time copy of the data.
+
+== Limitations for creating volume clones in multi-node topology
+
+{lvms} has the following limitations for creating volume clones in multi-node topology:
+
+* Creating volume clones is based on the LVM thin pool capabilities.
+* The node must have additional storage after creating a volume clone for further updating the original data source.
+* You can create volume clones only on the node where you have deployed the original data source.
+* Pods relying on the PVC that uses the clone data can be scheduled only on the node where you have deployed the original data source.
+* If the PVC is not present and available before creating a volume clone, the scheduler in multi-node {product-title} clusters does not provision the volume clones on the node where you have deployed the original data source.
```

Comment:
I tested this scenario on OCP multi-node cluster and doesn't happen to be valid. 

I created a Pod-1 resource with PVC resource named 'pvc-clone' which is not yet created on cluster. Pod goes into pending state as expected, then I created a volume clone PVC resource named 'pvc-clone' in which I have set dataSource as already existing bound PVC named 'ori-pvc'. Now Pod-1 goes into 'Running' state and when I checked, both source PV and cloned PV are mounted on same worker node.
@jakobmoellerdev Could you take a look if above scenario/statement is valid or not?

Reply:
The use case description is not quite valid:
The flow to reproduce is:
1. Create PVC (A) that is not bound yet
2. Create PVC (B) with dataSource set to other PVC (A) that is not bound
3. Create Pod (B) that should consume PVC (B)
4. Create Pod (A) that should consume PVC (A)

Now it can happen that the scheduling decision for Pod (B) is set to a different node than Pod (A) and thus scheduling failure occurs. This does not happen every time though so this might need to be retried a few times.

Reply:
@jakobmoellerdev  I reproduced these steps, Pod-A and Pod-B were deployed on same node as expected. And, PVC-A and PVC-B had different values for `volume.kubernetes.io/selected-node`. Whereas, both source PV and clone PV were mounted on same node where Pod-A & Pod-B were deployed.

But as per the statement on line -19, we are saying that "scheduler in multi-node OCP, source volume and volume clone will not be provisioned on the same node", which is not same as per the above results. WDYT?

Reply:
Ahh I think this aligns with my observations in https://issues.redhat.com/browse/OCPEDGE-148 and I had the same observation because TopoLVM ignores the scheduling decision in favor of its own topology. I suggest dropping the note due to this.

Reply:
Thanks for confirming again @radeore ! @sr1kar99 I think we should drop the part

"If the PVC is not present and available before creating a volume clone, the scheduler in multi-node {product-title} clusters does not provision the volume clones on the node where you have deployed the original data source."